src/services/pagination/Paginator.js

```diff
@@ -0,0 +1,205 @@
+import LesgoException from '../../exceptions/LesgoException';
+
+const FILE = 'Services/pagination/Paginator';
+
+export default class Paginator {
+  /**
+   * Constructor
+   *
+   * @param db
+   * @param sql
+   * @param sqlParams
+   * @param perPage
+   * @param currentPage
+   */
+  constructor(db, sql, sqlParams, perPage, currentPage = null) {
```

Comment:
@lordmangila @mosufy we can make the perPage by default as 10 and currentPage to 1, similar in laravel length aware.

Reply:
Hello @daison12006013 I now have applied the suggestion :)